### ISSUE TITLE
2.7.0 (stable) minor release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2580]
- #2435
- #2577 
- #2571 

## Pick list
```
git cherry-pick 4fced3536779b4cbae5c9e9c90776b31c2ae450e
git cherry-pick 2b36da9404da61e4700171530dfaf5c801bb68d5
git cherry-pick 516d315f95050639a08ae94501d818102459c35f
```

**Comment by joeparsons** [comment]
# Release notes snippet
```
_**Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported._

_To update to a Quickstart 2.7.x release (e.g. 2.7.0) from a version less than 2.6.8 you must first update to 2.6.8 and run database updates.  Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x and some core modules included in Drupal 9 have been removed in Drupal 10._

## New changes since 2.7.0-rc3
- Closes #2433 Add users to ManualMigrationLookup #2435
- Closes #2576 Line breaks converted to HTML in embedded entities #2577
- Closes #2570 Adding update to install az_curated_views #2571

## All changes

### New changes (not included in 2.6.x)
#### PHP Versions
Drupal 10 drops support for PHP 7.4 and 8.0. 
The minimum supported version of PHP is now 8.1

#### New modules
- [Flag](https://www.drupal.org/project/flag)
- [Asset Injector](https://www.drupal.org/project/asset_injector)
- [Media Entity File Replace](https://www.drupal.org/project/media_entity_file_replace)

#### Deprecated modules slated for removal
- [Entity Browser](https://www.drupal.org/project/entity_browser)

#### New feature
- Closes #2383 Upgrade tag page view. #2406
- Closes #2366 Allow linking images that are embedded in content. #2480
- Fixes #2247 Publication BibTeX Import #2358
- Closes #2380 Adding permissions for using shortcuts. #2481
- Fixes #2440 Publication Translations #2461
- Closes #2383 Enable az_digital_asset_library by default #2484
- Closes #2162, Closes #2477 Update drupal/intelligencebank from 3.0.0 to 4.0.0. #2163
- Fixes #2255, Fixes #2299 API Endpoint for News Export #2262
- Fixes #2359 Add Curated Publication Listings. #2470
- Fixes #2439 Add Publication Image to Full View Display and Card View Display #2467
- Closes #2289: Add basic linkchecker extractor for az_card, and az_accordion field types. #2291
- Closes #2431 Allow content administrators to manage their own url aliases. #2432
- Closes #2361 - Add multilingual support #2390
- Closes #2091, #1605, #1772 Add an optional thumbnail image field to AZ News, AZ Event #2308
- Closes #2375 Adding Flag module to Quickstart with Basic Permissions #2376
- Closes #2395 - Enable Splitscreen by default #2396
- Closes #2106 Allow \<code\> and/or \<pre\> HTML elements in az_standard text format #2393
- Fixes #2313 Update Citation Style Configuration #2316
- Closes #2368 Add option to move Sidebar First region below page content on mobile #2369
- Closes #1528: Allow users to replace files, also rename files through re-upload #2311
- Closes #2502: Update Analytics Dashboard link in toolbar. (#2511)
- Closes #2300 Enable Search Keywords view filter (#2544)

#### Bug fix
- #2403: Refactor AllowDrupalLogin test trait to not override parent::setup(). #2411
- Closes #2363: Use arm64-compatible webdriver image for lando. #2364
- Closes #2495 Menu blocks appearing on pages that they should not appear on (#2500)
- Fixes #2512 Cursor missing in new text area on Chrome (#2513)
- Fixes #2509 Content editor role unintentionally deleted. (#2510)
- Closes #2518 RC2 Bug Fixes introduced extra space to the main navigation (#2520)
- Closes #2525 Add error handling to 2.7.x az_news database update. (#2526)
- Closes #2576 Line breaks converted to HTML in embedded entities #2577
- Closes #2570 Adding update to install az_curated_views #2571

#### Breaking or visual change to existing behavior
- Closes #2473: Update display of links and CV fields in default Person display #2474
- Closes #1396: Added 3-col view display for AZ Person #2381
- Closes #2503: Update view display for AZ Person (#2501)
- Closes #2504 News articles are not displaying thumbnail images on view displays (#2514)

#### Drupal core major level update
- Closes #2081: Upgrade Drupal core to 10.0.9 #2354

#### Drupal contrib project updates
- Update drupal/entity_embed requirement from 1.3.0 to 1.4.0 #2468
- Update bower-asset/bootstrap-datepicker requirement from 1.9.0 to 1.10.0 #2410
- Update drupal/ckeditor_bs_grid requirement from 2.0.8 to 2.0.9 #2428
- #2213: Update drupal/migrate_queue_importer requirement from 2.0 to 2.1.0 #2445
- Update citation-style-language/styles requirement from 0.0.199 to 0.0.237 #2399
- Update drupal/draggableviews requirement from 2.1.1 to 2.1.3 #2372
- Update drupal/link_class requirement from 2.0.0 to 2.1.0 #2388
- Update drupal/media_migration requirement from 1.0.0-alpha14 to 1.0.0-alpha16 #2248
- Closes #2268: Add drupal/asset_injector as optional module. #2269
- Update citation-style-language/styles requirement from 0.0.237 to 0.0.249 #2427
- Update drupal/quick_node_clone requirement from 1.15 to 1.16.0 #2290
- Update drupal/search_exclude requirement from 2.0.0-beta2 to 3.0.0-beta1 #2193
- Update drupal/inline_entity_form requirement from 1.0-rc14 to 1.0.0-rc15 #2279
- Update drupal/linkit requirement from 6.0.0-beta3 to 6.0.0-beta4 #2280
- #2213: Update drupal/migrate_tools requirement from 5.2.0 to 6.0.1 #2450
- Update drupal/google_tag requirement from 1.6.0 to 2.0.2 #2337
- Closes #2442: Update drupal/bootstrap_barrio from 4.30 to 5.1.10. #2452
- Update drupal/better_exposed_filters requirement from 5.0 to 6.0.3 #1999
- Update bower-asset/bootstrap-datepicker requirement from 1.9.0 to 1.10.0 #2410
- Update drupal/media_migration requirement from 1.0.0-alpha14 to 1.0.0-alpha16 #2248
- Update drupal/linkit requirement from 6.0.0-beta4 to 6.0.0-rc1 #2336
- Update drupal/link_class requirement from 2.0.0 to 2.1.0 #2388
- Update drupal/draggableviews requirement from 2.1.1 to 2.1.3 #2372
- Closes #2321: Update drupal/menu_block requirement from 1.7.0 to 1.10.0 #2325
- Update drupal/chosen requirement from 3.0.5 to 4.0.0 #2333
- Update drupal/asset_injector requirement from 2.16.0 to 2.17.0 #2397
- Closes #2499: Add viewsreference patch to address Ajax failures. (#2505)
- Update drupal/ckeditor requirement from 1.0.1 to 1.0.2 (#2496)
- Closes #2516: Apply patch for XML sitemap D10 incompatibility issue. (#2517)
- Update drupal/inline_entity_form requirement from 1.0.0-rc15 to 2.0.0-rc4 (#2448)
- Closes #2552 Cannot remove instances of recurring events; Update drupal/smart_date requirement from 3.6.1 || 3.7.2 to 4.0.2 (#2494)

#### Other
- Closes #2119 Add categories to events and pages in demo content #2244
- Closes #2245 Unused imports in flexible page settings form #2246
- Update composer branch alias in preparation for 2.7.0 pre-release. #2492
- 2.7.x default branch changes #2493
- #2331 Update slack channel names from #azdigital-general to #azdigital-support (#2507)
- #2330 Update slack channel names from #friday-meetings to #azdigital-meetings (#2508)
- Update Arizona Bootstrap to 2.0.24 (#2522)
- #2560 Update github action to use specific release branch (2.7.x) of scaffolding repo (#2563)
- Closes #2529: Remove DB updates added before 2.6.0. (#2531)
- Closes #2491: Re-exported configuration files for 2.7.x (#2542)

### Backported changes (also included in 2.6.x)
#### New feature
- Closes #2433 Add users to ManualMigrationLookup #2435

#### Bug fix
- Closes #2371: Transform IDs in both source and destination paths for migrated redirects. #2373
- Fixes #2378 Academicpuma/locales abandoned #2379
- Closes #2315: Downgrade menu_block to 8.x-1.7 until we have a better solution for empty sidebars getting rendered. #2320
- Fixes #2312 Changes to existing news, event and person content do not update without clearing cache. #2318
- Closes #2264 Migrate path aliases for nodes if they exist, otherwise turn on Pathauto. #2281
- Fixes #2277 Text Area, and Single Card paragraphs behavior settings migrating incorrectly. #2278
- Fixes #2282 News, Event, Person external links do not migrate. #2283
- Closes #2288 Fix broken az_barrio footer logo field validation #2292
- Closes #2259 use null coallesce operator when creating strings to print from deeply nested array values (ParagraphsExtraInfoFieldMerge). #2260
- Closes #2253 - Add az_flexible_page to smart_title settings #2254
- Closes #2249: Fix migration config for link field on news and events that is generating PHP warnings. #2250
- Closes #2251 Remove block_class config for az_barrio_search block and add az-search-block class via preprocess hook. #2252
- Closes #2227 re-add linked image in az_person az_row view mode. #2228
- Closes #2229: Ensure block_class module is installed. #2232
- Fixes #2362 Publication Views exposed filters overlap #2469
- Closes #2497 Reorder people row weight input boxes don't show values (#2498)

#### Experimental module feature
- Closes #2222: Require PHP 8.0+ for experimental Trellis Events integration module. #2223
- Fixes #2214 Add a View to Publications to have all searches in one View #2231
- Fixes #2233 Make Publication citation style not dependent on View Mode #2236
- Closes #2273: Fix az_events_trellis coding standards issues. #2274
- Closes #2417: Fix double-encoded publication file URLs. #2430
- Closes #2239: Remove unnecesssary timezone processing from Trellis Events importer. #2241

#### Drupal core - Patch level release
- Update drupal/core-recommended requirement from 9.5.8 to 9.5.9 #2391
- Closes #2348: Update Drupal core to 9.5.8 for SA-CORE-2023-005 #2349
- Update drupal/core-recommended requirement from 9.5.5 to 9.5.7 #2306
- Closes #2294: Update Drupal core to 9.5.5 (moderately critical security update). #2297
- Update drupal/core-recommended requirement from 9.5.3 to 9.5.4 #2266

#### Drupal contrib - Patch level release
- Closes #2464 Change source of bootstrap-datepicker from bower-asset to npm-asset #2465
- Closes #2458 Update az_carousel required modules: slick, slick_views, blazy. #2459
- Closes #2382 - Fix PHP 8 code deprecation in custom smart_date migrate plugin. #2444
- Closes #2286 Update allowed Smart Date 3.7.x version from 3.7.0 to 3.7.2 #2287
- #2234: Allow smart_date 3.6.1 to be used since 3.7.0 requires PHP 8.0+. #2238
- Update drupal/workbench_access requirement from 2.0.0 to 2.0.1 #2230
- Closes #2234: Update drupal/smart_date from 3.6.1 to 3.7.0. #2235

#### Other
- Documentation: Update various pre-migration suggestions to include checking pathauto settings #2285
- Documentation: Add pre-migration task to menu links migration docs. #2284
- Update composer branch alias in preparation for 2.6.0 pre-release. #2218
- Close #2424 Adding badge display for all default taxonomy vocabularies. #2425
- Closes #2437 - Fix Slack channel and meeting information in CONTRIBUTING.md #2438
- Closes #2341: Enable use of description on Person's CV field #2343
- Closes #2265 Add manual migration lookup process plugin for migrate. #2271
- Fixes #2257, Fixes #2258 Publication Person Block Title, Add Publication Taxonomy #2270
- Closes #2342: Change links to chili on Event displays #2344
- Closes #2543 Retire theme-debug lando/ddev tooling. (#2545)
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.0
